monkey/monkey_island/cc/services/attack/attack_config.py

```diff
@@ -0,0 +1,160 @@
+    """
+    Applies ATT&CK matrix to the monkey configuration
+    :return:
+    """
+    attack_techniques = get_technique_values()
+    monkey_config = ConfigService.get_config(False, True, True)
+    monkey_schema = ConfigService.get_config_schema()
+    set_arrays(attack_techniques, monkey_config, monkey_schema)
+    set_booleans(attack_techniques, monkey_config, monkey_schema)
+    ConfigService.update_config(monkey_config, True)
+
+
+def set_arrays(attack_techniques, monkey_config, monkey_schema):
+    """
+    Sets exploiters/scanners/PBAs and other array type fields in monkey's config according to ATT&CK matrix
+    :param attack_techniques: ATT&CK techniques dict. Format: {'T1110': True, ...}
+    :param monkey_config: Monkey island's configuration
+    :param monkey_schema: Monkey configuration schema
+    """
+    for key, definition in monkey_schema['definitions'].items():
+        for array_field in definition['anyOf']:
+            # Check if current array field has attack_techniques assigned to it
+            if 'attack_techniques' not in array_field:
+                continue
+            try:
+                should_remove = not should_enable_field(array_field['attack_techniques'], attack_techniques)
+            except KeyError:
+                # Monkey schema field contains not yet implemented technique
+                continue
+            # If exploiter's attack technique is disabled, disable the exploiter/scanner/PBA
+            r_alter_array(monkey_config, key, array_field['enum'][0], remove=should_remove)
+
+
+def set_booleans(attack_techniques, monkey_config, monkey_schema):
+    """
+    Sets boolean type fields, like "should use mimikatz?" in monkey's config according to ATT&CK matrix
+    :param attack_techniques: ATT&CK techniques dict. Format: {'T1110': True, ...}
+    :param monkey_config: Monkey island's configuration
+    :param monkey_schema: Monkey configuration schema
+    """
+    for key, value in monkey_schema['properties'].items():
+        r_set_booleans([key], value, attack_techniques, monkey_config)
+
+
+def r_set_booleans(path, value, attack_techniques, monkey_config):
+    """
+    Recursively walks trough monkey configuration (DFS) to find which boolean fields needs to be set and sets them
+    according to ATT&CK matrix.
+    :param path: Property names that leads to current value. E.g. ['monkey', 'system_info', 'should_use_mimikatz']
+    :param value: Value of config property
+    :param attack_techniques: ATT&CK techniques dict. Format: {'T1110': True, ...}
+    :param monkey_config: Monkey island's configuration
+    """
+    if isinstance(value, dict):
+        dictionary = {}
+        # If 'value' is a boolean value that should be set:
+        if 'type' in value and value['type'] == 'boolean' and 'attack_techniques' in value:
+            try:
+                set_bool_conf_val(path, should_enable_field(value['attack_techniques'], attack_techniques), monkey_config)
```

Comment:
PEP8, break into multiple lines
```suggestion
                set_bool_conf_val(path, should_enable_field(value['attack_techniques'], attack_techniques),
                                  monkey_config)
```